src/schedule.js

```diff
@@ -498,6 +515,20 @@ function scheduleJob() {
   return null;
 }
 
+function rescheduleJob(job, spec) {
+  var success = false;
+  if (job instanceof Job) {
+    success = job.reschedule(spec);
+  } else {
+    var findJob = getJob(job);
+    if (findJob instanceof Job) {
+      return rescheduleJob(findJob, spec);
+    }
+  }
+
+  return success;
+}
+
```

Comment:
How about this function returns the `Job` instance in the same way as `scheduleJob` does?